lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -46,13 +46,7 @@
         </button>
     <% } else {%>
         <div class="no-action-message">
-            <%- gettext('Coming Soon') %>
-        </div>
-        <div class="enrollment-opens">
-            <%- gettext('Enrollment Opens on') %>
-            <span class="enrollment-open-date">
-                <%- enrollment_open_date %>
-            </span>
+            <%- gettext('Not Currently Available') %>
```

Comment:
@rlucioni Got it. I don't doubt that most, if not all, of the courses created is leaving the enrollment open date to blank. It's my guess that for edx.org, the course created and published doesn't mean the course is visible to students. We still need the course meta information duplicated in edx-mktg at the moment. However, with our program details page, the aforementioned presumed understanding by course staff should change.
I am fine with removing the "coming soon" case in this PR. I just want to make sure all the scenarios are handled properly by our software.